zbus/src/connection/socket/unix.rs

```diff
@@ -102,7 +102,7 @@ impl super::WriteHalf for Arc<Async<UnixStream>> {
     }
 
     async fn peer_credentials(&mut self) -> io::Result<crate::fdo::ConnectionCredentials> {
-        get_unix_peer_creds(self).await
+        super::ReadHalf::peer_credentials(self).await
```

Comment:
Thanks for splitting this into its own commit. Could you please also add the rationale/short description in the commit for this change?

Reply:
wasn't resolved.

Reply:
Oops. Should be now.